xknx/remote_value/remote_value.py

```diff
@@ -110,13 +110,13 @@ def to_knx(self, value):
 
     async def process(self, telegram, always_callback=False):
         """Process incoming or outgoing telegram."""
-        if not self.has_group_address(telegram.group_address):
+        if not self.has_group_address(telegram.destination_address):
```

Comment:
will this raise TypeError if destination_address is a PhysicalAddress ? Is this tested?